.gitignore

```diff
@@ -144,11 +144,13 @@ credentials.csv
 .bloop
 
 # python tests
+*.bin
 demo/**/*.txt
 *.dmatrix
 .hypothesis
 __MACOSX/
 model*.json
+/tests/python/models/models/
```

Comment:
Noticed some model files left behind from running all the Python tests locally while developing this. These `.gitignore` rules prevent checking them into source control.